### PR TITLE
[#43153] Fix storage admin breadcrumbs

### PR DESCRIPTION
**Pull request by apfohl**
https://community.openproject.org/work_packages/43153

### CODE REVIEW

File: modules/storages/app/views/storages/admin/storages/new_oauth_client.html.erb [hunk 1]
```diff
@@ -11,6 +11,7 @@
                      size: 40,
                      container_class: '-wide' %>
     <span class="form--field-instructions">
+      <%= "#{t("storages.instructions.copy_from")}: " %>
       <%= "#{t("storages.instructions.copy_from")}: " %>
```
Comment (HDinger) [inline modules/storages/app/views/storages/admin/storages/new_oauth_client.html.erb]: This is doubled

File: modules/storages/app/views/storages/admin/storages/show_oauth_application.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- Standard Ruby view, please see the controller for comments -->
 <% html_title t(:label_administration), t("project_module_storages"), @object.name, "#{t("storages.provider_types.#{@object.provider_type}.name")} #{t("storages.label_oauth_application_details")}" %>
-<% local_assigns[:additional_breadcrumb] = "#{t("storages.provider_types.#{@object.provider_type}.name")} #{t("storages.label_oauth_application_details")}" %>
-<%= toolbar title: "#{t("storages.provider_types.#{@object.provider_type}.name")} #{t("storages.label_oauth_application_details")}" %>
+<% local_assigns[:additional_breadcrumb] = %W[#{@object.name} #{t("storages.label_oauth_breadcrumb.openproject")}] %>
```
Comment (HDinger) [inline modules/storages/app/views/storages/admin/storages/show_oauth_application.html.erb]: Don't know the requirements here, is it correct to always refer to `openproject`?

**Review by HDinger — COMMENTED** [review]
It is a bit hard for me to review this, as the ticket is lacking basically every Information except the title. So I can only guess what to test exactly. 🤷‍♀️ 

Looking at the admin breadcrumbs, I found two issues: 

- [x] On `/admin/settings/storages`, the last part of the breadcrumb is a link. Usually, we don't link the current page in the breadcrumb. 

Bildschirmfoto 2022-07-14 um 11 26 57

- [x] On `/admin/settings/storages/1/oauth_client/new`, there is a link missing to "Nextcloud"
Bildschirmfoto 2022-07-14 um 11 30 53

File: modules/storages/app/views/storages/admin/storages/show_oauth_application.html.erb (hunk 2 above)
Comment (apfohl) [reply modules/storages/app/views/storages/admin/storages/show_oauth_application.html.erb]: Yes, this view will always show a created OAuth application which is created in OpenProject. Only the preceding view can be different depending on the type of the attached storage.

File: modules/storages/app/views/storages/admin/storages/new_oauth_client.html.erb (hunk 1 above)
Comment (apfohl) [reply modules/storages/app/views/storages/admin/storages/new_oauth_client.html.erb]: Fixed.

**Review by HDinger — APPROVED** [review]
Please rebase to the `release/12.2` branch and change the target in the PR. After that, it is good to go from my side 👍

**Pull request merged**